packages/twenty-e2e-testing/tests/workspaces.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { test, expect } from '@playwright/test';
+import { sh } from '../drivers/shell_driver';
+
+const date = new Date();
+
+test.afterEach(async ({ page, browserName }) => {
+  await page.screenshot({
+    path: `./packages/twenty-e2e-testing/results/screenshots/${browserName}/${date.toISOString()}.png`,
+  });
+});
+
+test.describe('', () => {
+  test('Testing logging', async ({ page }) => {
+    await page.goto('/');
+    await page.getByRole('button', { name: 'Continue With Email' }).click();
+    await page.getByPlaceholder('Email').fill('tim@apple.dev');
+    await page.getByRole('button', { name: 'Continue' }).click();
+    await page.getByPlaceholder('Password').fill('Applecar2025');
+    await page.getByRole('button', { name: 'Sign in' }).click();
+    await page.getByRole('link', { name: 'Opportunities' }).click();
+    await expect(page.locator('tbody > tr')).toHaveCount(4);
+    expect(page.url()).not.toContain('/welcome');
+  });
+
+  test('Creating new workspace', async ({ page, browserName }) => {
+    // this test must use only 1 browser, otherwise it will lead to success and fail (1 workspace is created instead of x workspaces)
```

Comment:
Shouldn't we instead play the whole test on each brother while resetting the DB before each ?

Reply:
Ok let's use only chromium as we said earlier then ?